src/pages/PullRequestPage/Dropdowns/PullBundleDropdown.spec.tsx

```diff
@@ -139,7 +139,7 @@ describe('BundleMessage', () => {
             wrapper,
           })
 
-          const title = await screen.findByText('Bundle Report:')
+          const title = await screen.findByText('Bundle report:')
```

Comment:
Is there a ticket where we're keeping all the copy/markup and general values that we want to keep consistent bw the UI and the PR comment? Would be useful to ensure those are aligned 👌 thinking of this and the KB/kB thing

Reply:
Just the figma file. The kB KB issue may be that the designs may need to be updated to be aligned with the international standards, I'm not manually writing that in the UI and just using the Intl utility.